### PR TITLE
Update greater_common_divisor.py

### PR DESCRIPTION
**Pull request by madforpython**
Check This code.It's beginner friendly.

### CODE REVIEW

File: maths/greater_common_divisor.py [hunk 1]
```diff
@@ -1,25 +1,25 @@
-"""
-Greater Common Divisor.
+# Python program to find the G.C.D of two input number
 
-Wikipedia reference: https://en.wikipedia.org/wiki/Greatest_common_divisor
```
Comment (cclauss) [inline maths/greater_common_divisor.py]: Why should we take this away??

File: maths/greater_common_divisor.py [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
-"""
+# define a function
+def computeGCD(x, y):
```
Comment (cclauss) [inline maths/greater_common_divisor.py]: What about __greatest_common_denominator()__??

File: maths/greater_common_divisor.py [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
 
+# choose the smaller number
+    if x > y:
+        smaller = y
+    else:
+        smaller = x
```
Comment (cclauss) [inline maths/greater_common_divisor.py]: __smaller = min(x, y)__

File: maths/greater_common_divisor.py (hunk 1 above)
Comment (madforpython) [reply maths/greater_common_divisor.py]: Sharing My Experience.When i was at my initial Learning stage i used to face several difficulties undersatnding code which contains complexities.So i prefer keeping codes ShortnSweet.

File: maths/greater_common_divisor.py (hunk 2 above)
Comment (madforpython) [reply maths/greater_common_divisor.py]: It's Greatest Common Divisor Code

File: maths/greater_common_divisor.py (hunk 3 above)
Comment (madforpython) [reply maths/greater_common_divisor.py]: as I mentioned above.I like to keep codes beginner friendly.

**Pull request closed without merging**